### ISSUE TITLE
Run ktistec on a subdomain

### ISSUE THREAD

**Issue by sean-clayton** [toddsundsted/ktistec#85]
Is ktistec made to run on a subdomain, but "live" on the domain name? Like [how microblog.pub does it](https://docs.microblog.pub/installing.html#(advanced)-running-on-a-subdomain). Eg, running on `ktistec.example.com` but posts and such appear as `example.com`?

**Comment by toddsundsted** [comment]
@sean-clayton i don't think so, although i admit that i don't understand the use case. does microblog.pub act like a general purpose webfinger server for the domain then?

**Comment by sean-clayton** [comment]
> @sean-clayton i don't think so, although i admit that i don't understand the use case. does microblog.pub act like a general purpose webfinger server for the domain then?

Pretty much. I believe this is more of a "vanity" thing where your ActivityPub posts look 'nice' with just example.com showing.

[This blog post](https://calebhearth.com/running-mastodon) shows another example (using mastodon, though).
